README.md

```diff
@@ -117,6 +117,42 @@ This is an example response body:
 }
 ```
 
+### ADD - Adding draft editorial annotations and writing them in PAC
+
+Using curl:
+
+```
+curl http://localhost:8080/draft/content/{content-uuid}/annotations -X POST --data '{
+        {
+          "id": {concept-uuid},
+          "predicate": {predicate}
+        }
+}'
+```
+
+An ADD request on this endpoint adds an annotation to the editorially curated published annotations for a specific piece of content. To retrieve these annotations it calls [UPP Public Annotations API](https://github.com/Financial-Times/public-annotations-api) using the "lifecycle" parameter.
```

Comment:
We need `POST` request here instead of `ADD`. Also it will be good if we can clarify that we are saving draft annotations and they will override any unpublished draft annotations for the same piece of content.

Reply:
Fixed